config/webhook/manifests.yaml

```diff
@@ -0,0 +1,56 @@
+
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
```

Comment:
@ishitasequeira @anandf  why do we need to introduce a mutating webhook in addition to the validation webhook?

Reply:
You are right.. I will remove the Mutating Webhook.